src/main/scala/beam/utils/plan/SubSamplerApp.scala

```diff
@@ -0,0 +1,83 @@
+package beam.utils.plan
+
+
+import java.io.{BufferedInputStream, File, FileInputStream}
+import java.nio.file.Paths
+import java.util.Locale
+import java.util.zip.GZIPInputStream
+
+import org.matsim.core.config.ConfigUtils
+import org.matsim.core.network.io.MatsimNetworkReader
+import org.matsim.core.population.io.{StreamingPopulationReader, StreamingPopulationWriter}
+import org.matsim.core.scenario.ScenarioUtils
+import org.matsim.core.utils.geometry.transformations.GeotoolsTransformation
+import org.matsim.core.utils.io.UnicodeInputStream
+
+import scala.util.Try
+
+
+object SubSamplerApp extends App {
+
+  def createSample(inputPopulationFile: File, networkFile: File, samplesize: Double, outputPopulationFile: File): Unit = {
```

Comment:
should we pass the whole config file or more files and output also more files, e.g. PopulationAttributes, HouseholdAttributes, populaiton, housholds, vehicles, etc.

Reply:
updated